constructor/osx/prepare_installation.sh

```diff
@@ -39,9 +39,10 @@ touch "$PREFIX/conda-meta/history"
 # but we haven't created $PREFIX/pkgs yet... do it in a temporary location
 # shellcheck disable=SC2050
 if [ "__VIRTUAL_SPECS__" != "" ]; then
-    notify 'Checking virtual specs compatibility: __VIRTUAL_SPECS__'
+    notify "Checking virtual specs compatibility: __VIRTUAL_SPECS__"
     CONDA_SOLVER="classic" \
     CONDA_PKGS_DIRS="$(mktemp -d)" \
+    SYSTEM_VERSION_COMPAT=0 \
```

Comment:
Since checking for the virtual package is removed, I don't think this variable is needed anymore.

Reply:
It doesn't hurt either I guess?